<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ACSP22 P05 - Formoe Mygind</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="btn_check">
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="line_state">
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
       <pointsize>13</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_seq">
     <property name="text">
      <string>Sequence:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="line_seq">
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
       <pointsize>13</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btn_state">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_state">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>State:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btn_seq">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="display_seq">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>480</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>qt/pim_sequence.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="display_state">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>5</y>
        <width>480</width>
        <height>700</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>qt/pim_state.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="dialogue">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>135</width>
        <height>40</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>5</y>
        <width>135</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
